### ISSUE TITLE
Providing trajectories in the leaderboard

### ISSUE THREAD

**Issue by shuyanzhou** [ServiceNow/AgentLab#193]
Many agent benchmarks require the participants to upload the execution trajectories to ensure the transparency and reproducibility. Would it be interesting to support this in BrowserGym leaderboard? Would it be possible to share the execution trajectories for the existing models for broader analysis? 

Examples: 
[SWE-bench](https://www.swebench.com/)
[WebArena](https://docs.google.com/spreadsheets/d/1M801lEpBbKSNwP-vDBkC_pF7LdyGU1f_ufZb_NWNBZQ/edit?gid=0#gid=0)

Thanks!

**Comment by RohitP2005** [comment]
@stegel  I am looking forward to work on this

**Comment by recursix** [comment]
Yes, we have an open ticked for doing exactly that. I hope we can get that done soon.

**Comment by ThibaultLSDC** [comment]
Hello @shuyanzhou, while we don't have automatic trace upload yet, we uploaded the runs from our latest experiments and made them available to the public [here](https://huggingface.co/datasets/agentlabtraces/agentlabtraces). :)
